config/routes.rb

```diff
@@ -17,6 +17,13 @@
   get 'register', to: "users#new"
   resources :users, only: [:show, :create]
 
+  get 'forgot_password', to: 'forgot_passwords#new'
+  resources :forgot_passwords, only: [:create]
```

Comment:
I am not sure if this is a resource. I personally prefer to make resources only where it is model backed.